aspnetcore/mvc/views/tag-helpers/built-in/anchor-tag-helper.md

```diff
@@ -66,6 +65,22 @@ If no `asp-controller` attribute is specified, the default controller calling th
  
 If the attribute `asp-action` is `Index`, then no action is appended to the URL, leading to the default `Index` method being called. The action specified (or defaulted), must exist in the controller referenced in `asp-controller`.
 
+### asp-page
+
+Use the `asp-page` attribute in an anchor tag to set its URL to point to a specific page. Prefixing the page name with a forward slash "/" creates a relative URL. The URL in the sample below points to the "Speaker" page in the current directory.
```

Comment:
~Use~
Use a ref style her. Get rid of the "you"s.
`/` is not a relative, `./` is relative - right?

Reply:
Show an example with route values - like on https://docs.microsoft.com/en-us/aspnet/core/mvc/razor-pages/

<a asp-page="./Edit" asp-route-id="@contact.Id">edit</a>

Reply:
You can't use asp-route with it, it generates a querystring in the URL.

Reply:
Look at the examples on  https://docs.microsoft.com/en-us/aspnet/core/mvc/razor-pages/